src/_components/Beeswarm.html.svelte

```diff
@@ -19,7 +19,7 @@
 	/** @type {Number} [spacing=1.5] - Spacing, in pixels, between each circle. */
 	export let spacing = 1.5;
 
-	/** @type {Function} [getTitle] — An accessor function to get the field on the data element to display as a hover label. Mostly useful for debugging, needs better styling for production. */
+	/** @type {Function|undefined} [getTitle] - An accessor function to get the field on the data element to display as a hover label. Mostly useful for debugging, needs better styling for production. */
```

Comment:
If a param is optional with the square brackets, doesn't that imply that it could be undefined?

Reply:
I probably wasn't consistent with this before since I see that's how I had the `getTitle` propr in Beeswarm.svelte